homepage/templates/faq.html

```diff
@@ -0,0 +1,62 @@
+
+                    <h2>There’s a feature I wish to see in Orange.</h2>
+                    <p>Please submit a feature request to our <a href="https://github.com/biolab/orange3/issues">issue tracker</a>. Describe what you wish to see in Orange and, if possible, provide a visual example. Tag the request with <b>enhancement</b>. If enough people seem to be interested in the request, we will try to implement it.</p>
+
+                    <p>Before submitting a new feature request, check if someoone has already requested the same feature. Upvote the request with a +1 and/or add your comment. Only consider opening a new request if you cannot find a corresponding existing request.</p>
+
+                    <h2>I would like to learn how to use Orange. Where can I find the resources?</h2>
+                    <p>Widget documentation available on our <a href="https://orange.biolab.si/toolbox/">website</a>. We also provide an e-mail course (<a href="https://biolab.us15.list-manage.com/subscribe?u=2cad1f1cbf3a532a77efe4354&id=6f257f15ff">subscribe here</a>), beginner-level <a href="https://www.youtube.com/playlist?list=PLmNPvQr9Tf-ZSDLwOzxpvY-HrE0yv-8Fy">YouTube tutorials</a> and a regularly updated <a href="https://blog.biolab.si/">blog</a>.</p>
+
+                    <h2>I have a question concerning data mining, statistics and machine learning algorithms.</h2>
+                    <p>Visit <a href="https://datascience.stackexchange.com/questions/tagged/orange">Data Science StackExchange</a> and ask the community!</p>
+
+                    <h2>What data does Orange support?</h2>
+                    <p>Core Orange supports Excel, comma- and tab-delimited files (.xlsx, .csv, .tab). It also reads online data, such as Google Spreadsheets. Orange3-ImageAnalytics add-on can import images (.jpg, .tiff, .png). Orange3-Corpus add-on can import text files (.txt, .docx, .odt). SQL widget supports Postgres and MySQL databases.</p>
+
+                    <h2>I want to use Orange on big data.</h2>
+                    <p>Orange provides SQL widget, which samples data instances and enables exploratory data analysis on large datasets. You need <a href="http://initd.org/psycopg/">psycopg2</a> module installed to be able to see the widget. Install it directly in Python with <code>pip install psycopg</code> command. You can also use <a href="https://www.2ndquadrant.com/en/resources/2uda/">2UDA package</a> for easier installation. Please note that Orange supports only Postgres and MySQL databases for now.</p>
+
+                    <h2>Can I export Orange workflow as a Python script?</h2>
+                    <p>Unfortunately, no. We've debated this long and hard and it is simply not possible to maintain interactive analysis and export the workflow as a Python script at the same time. And interactive visualizations and exploratory data analysis are the very heart of Orange.</p>
```

Comment:
Maybe a link to https://github.com/biolab/orange3/issues/1341?

Reply:
I would avoid "simply not possible".
A (limited) functionality of this type would probably be possible, but would be a very big project to do well and cost more than the expected benefits?